src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -683,15 +688,15 @@ local nativeShutdown = os.shutdown
 function os.shutdown()
     nativeShutdown()
     while true do
-        coroutine.yield()
+        nativeyield()
```

Comment:
Could consider setting a "neverResume" filter, or somesuch.

Reply:
Ideally `os.shutdown` would just terminate/suspend the VM there, meaning we didn't need this code at all.